profile/src/main/java/com/scottlogic/deg/profile/reader/names/NameCSVPopulator.java

```diff
@@ -0,0 +1,32 @@
+package com.scottlogic.deg.profile.reader.names;
+
+import org.apache.commons.csv.CSVFormat;
```

Comment:
Does this introduce a dependency that we need to ensure gets included in the fat jar as part of the build process?

Reply:
No. The fat jar job automatically places all the dependencies into the jar.